report/sarif/builder.go

```diff
@@ -1,5 +1,7 @@
 package sarif
 
+import "github.com/securego/gosec/v2"
```

Comment:
Please don't bring any gosec dependency in here

Reply:
Totally agree. So did you mean I should put `NewSuppression` in `sarif/builder.go` but should not import gosec?

Reply:
The SupressionInfo type is not logically related to the format (e.g. Sarif) in this case. In the same time, we tried to avoid importing the gosec package in here. All the types of the variable in the method are primitive for a reason. 

I would rather create a duplicated type in the formatter (e.g. Sarif) and populate it at the time when the method gets called by coping the data from `gosec.SuppressionInfo`.

Reply:
I mean that 
```go
func NewSuppression(kind string, justification string) *Suppression
```
goes in `sarif/builder.go`
while
```go
func buildSarifSuppressions(suppressions []gosec.SuppressionInfo) []*Suppression
```
stays in `sarif/formatter.go`